### ISSUE TITLE
[Bug] Non-md file with same name as md file styled as if it was the md file in file explorer

### ISSUE THREAD

**Issue by Trikzon** [mdelobelle/obsidian_supercharged_links#154]
If I have a md note with the same name as a non-md file (such as a pdf), then the non-md file will be styled the same way in the file explorer.

## Example
md note:
![image](https://user-images.githubusercontent.com/29845000/198865091-f90c1f74-16d0-4d46-94b7-973915d2f498.png)

pdf file:
![image](https://user-images.githubusercontent.com/29845000/198865095-250359cf-3a0d-4122-addf-2e2563f82d1f.png)

## Inconsistency
However, elsewhere the pdf file isn't styled the same as the md note. This inconsistency hints to me that this isn't intended behavior, so I'd like to see it made consistent. In my opinion, it should be fixed by making it so that the non-md note with the same name doesn't get styled in the file explorer.

### Example
In live preview, a link to the pdf isn't styled the same as the md note.
![image](https://user-images.githubusercontent.com/29845000/198865167-eb463cab-dedf-42d3-9896-19513df61cbd.png)

**Comment by HEmile** [comment]
Do you know what kind of rule activates here? Is it path based or tags/ property based?

**Comment by Trikzon** [comment]
All my rules are tag based.

For this file, I have a rule for tag `#source` that prepends a book emoji to the note's name.
I also have another rule for tag `#process/queue` that changes the note's color to a brown-ish color.

**Comment by HEmile** [comment]
I see, yes that's definitely not supposed to be styled like this then. I'll fix this in the next round.

**Comment by OptionalM** [comment]
I'd wager this is related to #163

**Comment by HEmile** [comment]
I agree. Please let me know if this is still an issue, otherwise I'll close it.
